analyzer/src/funTest/kotlin/Yarn2Test.kt

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Commit message: It's a bit unusual to have a foot note like `[1]` in the title. Better move it to the body text, after "documentation" or "Corepack".